### PR TITLE
Use brovider

### PR DESCRIPTION
**Pull request by ChaituVR**
Related https://github.com/snapshot-labs/snapshot/issues/2492

### CODE REVIEW

**Review by bonustrack — APPROVED** [review]

**Review by bonustrack — APPROVED** [review]

**Comment by ChaituVR** [comment]
Need a second review to merge 

@mktcode

**Review by mktcode — APPROVED** [review]

**Pull request merged**
